master/docs/examples/git_gerrit.cfg

```diff
@@ -185,6 +185,7 @@ def gerritSummaryCB(buildInfoList, results, status, arg):
 
 c['buildbotURL'] = 'http://buildbot.example.com/'
 c['status'].append(status.GerritStatusPush(gerrit_url, gerrit_user,
+                                           notify='ALL',
```

Comment:
If we want the user to use this as an example, I'd suggest to:
* add a comment what 'notify=ALL' means
* add a link that you put in the beginning of this PR